### ISSUE TITLE
Get rid of "You aborted the video playback"

### ISSUE THREAD

**Issue by sethborg** [videojs/video.js#1930]
I think you should let the user handle aborted video playback messages. Why? Because if you refresh the page or load a new page, your play will pop up this message which makes the user think something broke.

Try going to http://www.videojs.com/ and hit the play button. then refresh the page to see the message displayed over the player.

Workaround until this is fixed: .vjs-error .vjs-error-display { display: none; }

**Comment by sethborg** [comment]
OK This only seems to happen in Firefox. I imagine because once a page request is started it stops the html5 video player. I don't see them changing the way their browser works and this won't negatively affect the other browsers if you change this behavior so the user is responsible for the aborted message.

**Comment by heff** [comment]
I can't reproduce this in Firefox myself, but I don't doubt it happens in some cases. If you just use the video element itself (no video.js) what does it do? Firefox is actually the one browser that shows error text by default like video.js is doing, so it might do the same thing.

**Comment by mmcc** [comment]
All of the error messages are also customizable, so you change the wording if you feel it's too strong. You can also disable the error display altogether if you'd like: [example](http://jsbin.com/xiveyi/1/edit?html,output).

TL;DR, set `"errorDisplay": false` in your Video.js options.
